src/main/java/org/terasology/navgraph/NavGraphChunk.java

```diff
@@ -65,6 +72,10 @@ public NavGraphChunk(WorldProvider world, Vector3i chunkPos) {
         }
     }
 
+    /**
+     * Called when there is a change in terrain It re-marks all the floors within the NavGraphChunk
```

Comment:
Is this two sentences combined? If so, they should be on different lines. If not, it's missing some kind of punctuation between "terrain" and "It"